src/pages/CatInfoPage.js

```diff
@@ -66,7 +80,11 @@ export default function CatInfoPage() {
             ))}
           </UL>
         )}
-        {!catData && <p>Sorry, there is a problem. Please try again later. </p>}
+        {!catData && (
+          <CatDataFetchErrorMessage>
+            Sorry, there is a problem. Please try again later.{' '}
```

Comment:
```suggestion
            Sorry, there is a problem. Please try again later.
```

I guess that is not on purpose?

Reply:
thanks, i removed it